src/config/routes.config.tsx

```diff
@@ -0,0 +1,22 @@
+import { RouteObject } from "react-router-dom";
+import { LandingPage, TripDetailPage, TripsListingPage } from "../pages";
+import { ErrorPage } from "../pages/ErrorPage/ErrorPage";
+
+export const routesConfig: RouteObject[] = [
+  {
+    path: "/",
+    element: <LandingPage />,
+  },
+  {
+    path: "/trips",
+    element: <TripsListingPage />,
+  },
+  {
+    path: "/trips/:tripId",
+    element: <TripDetailPage />,
+  },
+  {
+    path: "*",
+    element: <ErrorPage />,
+  },
+];
```

Comment:
Create a "Layout" component with Header, Footer and the Component as "Outlet" instead of children. Add `ErrorPage` as ErrorElement.

```
{
element: <Layout />,
children: [
  {
      path: "/",
      element: <LandingPage />,
    },
    {
      path: "/trips",
      element: <TripsListingPage />,
    },
    {
      path: "/trips/:tripId",
      element: <TripDetailPage />,
    },
],
 errrorElement:<ErrorPage />
}
```

Reply:
Sure

Reply:
https://drive.google.com/file/d/1s2j65RohEHXdlGNd9wKHcNrAbhdK5Db4/view

This codebase was shared to us by Thejus during training. We can also use some parts of this code in our routing.

Reply:
Done in last commit